src/site/markdown/server/manage-webadmin.md

```diff
@@ -2496,6 +2497,45 @@ Response codes:
 
 Not implemented yet.
 
+## Deleted Messages Vault
+
+Deleted messages from an user will be stored in a restriction place where only delegated administrator can access.
+This place is called the Vault. Messages in the Vault will be deleted permanently by a configured retention policy.
+
+ - [Restore Deleted Messages](#Restore_deleted_messages)
+
+### Restore Deleted Messages
+
+An user can request to the administrator to restore his messages had been deleted and now at the Vault.
+To restore them, the administrator perform a http request:
+
+```
+curl -XPOST http://ip:port/deletedMessages/user/userToRestore@domain.ext
+```
+
+**All** messages from the user passed in the path param will be moved from the Vault to his new mailbox.
+Response code:
+
+ - 201: Task for restoring deleted has been created
+ - 400: Bad request, user parameter is invalid
+
+The scheduled task will have the following type `deletedMessages/restore` and the following `additionalInformation`:
+
+```
+{
+  "successfulRestoreCount": 47,
+  "errorRestoreCount": 0
+  "failRemovingCount": 19
+  "user": "userToRestore@domain.ext"
```

Comment:
no status?

Reply:
what do you mean status? status with me is the result of the task, either completed or failed, it's showed in the task detail, but this is the representation of task additional information.